frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -186,14 +186,16 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
 
   let resultDataVolume;
   if (source.requiresDatavolume()) {
-    resultDataVolume = DataVolumeWrapper.initializeFromSimpleData({
-      name: resultDataVolumeName,
-      storageClassName: storageClassName || null, // || null is to enable merging
-      type: source.getDataVolumeSourceType(),
-      size,
-      unit,
-      typeData: { name: pvcName, namespace, url },
-    })
+    resultDataVolume = new DataVolumeWrapper()
+      .init({
+        name: resultDataVolumeName,
```

Comment:
Do you mean to leave `name: resultDataVolumeName` and leave out the `generateName` property or to change the the `name` property to just use the existing name instead of combining it with the VM name?

Reply:
we should use the existing name if we are updating it

Reply:
Done!